widget/embedded/src/pages/SettingsPage.tsx

```diff
@@ -178,18 +189,52 @@ export function SettingsPage() {
         </Tooltip>
       </>
     ),
+    start: <DoneIcon color="gray" size={12} />,
     end: <Switch checked={infiniteApprove} />,
     onClick: toggleInfiniteApprove,
   };
 
+  const getThemeIcon = (theme: ThemeMode) => {
+    switch (theme) {
+      case 'auto':
+        return <AutoThemeIcon color="gray" size={12} />;
+      case 'dark':
+        return <DarkModeIcon color="gray" size={12} />;
+      default:
+        return <LightModeIcon color="gray" size={12} />;
+    }
+  };
+
+  const themeItem = {
+    id: 'theme-item',
+    type: <ListItem title="_" />,
```

Comment:
Why do we need this? `title="_"`

Reply:
I added this type to the items so that each item can have a unique component because the list component we had only took one type and we should have used the ListItem component instead of the ListItemButton here in the theme.